apps/formbricks-com/app/docs/contributing/gitpod/page.mdx

```diff
@@ -1,15 +1,97 @@
 import Image from "next/image";
+import GitpodPorts from "./gitpod-ports.webp";
+import GitpodAuth from "./gitpod-auth.webp";
+import GitpodNewWorkspace from "./gitpod-new-workspace.webp";
+import GitpodPreparing from "./gitpod-preparing.webp";
+import GitpodRunning from "./gitpod-running.webp";
 
 export const meta = {
   title: "Gitpod Setup",
   description:
     "With one click, you can setup the Formbricks developer environment in your browser using Gitpod",
 };
 
+#### Contributing
+
+# Gitpod
+
 ### One Click Setup
```

Comment:
Hey, can we make this a 2 ## heading so that it's parsed by the navigation bar as well?

Reply:
hey, thanks for the suggestion but we are using 1 # for heading in rest of the similar places in docs